src/systems/thruster/Thruster.hh

```diff
@@ -70,6 +70,15 @@ namespace systems
   ///                      defaults to 1000N]
   /// - <min_thrust_cmd> - Minimum thrust command. [Optional,
   ///                      defaults to -1000N]
+  /// - <wake_fraction>  - Relative speed reduction between the water
+  ///                      at the propeller vs behind the vessel.
+  ///                      [Optional, defults to 0.2]
+  /// - <alpha_1> - Constant given by the open water propeller diagram. Used
+  ///               in the calculation of the thrust coefficient.
+  ///               [Optional, defults to 1]
+  /// - <alpha_2> - Constant given by the open water propeller diagram. Used
+  ///               in the calculation of the thrust coefficient.
+  ///               [Optional, defults to 0]
```

Comment:
@marcoag bibliographic references here would be great.

Reply:
Added in https://github.com/gazebosim/gz-sim/pull/1652/commits/7351aab3928bb9c294b9c3df115b023f2c989098.